src/modules/ens/sagas.ts

```diff
@@ -183,10 +183,11 @@ function* handleFetchDomainListRequest(_action: FetchDomainListRequestAction) {
 }
 
 function* handleClaimNameRequest(action: ClaimNameRequestAction) {
-  const { ens, name } = action.payload
+  const { name } = action.payload
   try {
     const [from, eth]: [Address, Eth] = yield getCurrentAddress()
     const txHash = ''
+    const ens = {} as ENS
     console.log('handleClaimName', ens, name, eth)
```

Comment:
Make sense to continue with these `console.log`s at this point? Maybe we need to clean them from the our code.